app/src/main/java/com/example/kuit6_week1/MainActivity.kt

```diff
@@ -19,29 +37,208 @@ class MainActivity : ComponentActivity() {
+                    .size(60.dp)
+                    .border(
+                        color = Color.Black,
+                        width = 3.dp,
+                        shape = CircleShape
+                    )
+
+            )
+        }
+        SampleRow (
+            modifier = Modifier.padding(top = 20.dp)
+        ) {
+            Box(
+                modifier = modifier
+                    .size(76.dp)
+                    .border(
+                        width = 1.dp,
+                        color = Color.Black
+                    )
+                    .padding(8.dp)
+                    .background(Color.Red)
+            )
+            Box(
+                modifier = modifier
+                    .size(76.dp)
+                    .border(
+                        width = 1.dp,
+                        color = Color.Black
+                    )
+                    .padding(8.dp)
+                    .background(Color.Blue)
+                    .clip(CircleShape)
+            )
+        }
+        SampleRow (
+            modifier = Modifier.padding(20.dp)
+        ){
+            Image(
+                painter = painterResource( R.drawable.image),
+                contentDescription = "개 이미지",
+                modifier = Modifier.size(80.dp)
+            )
+
+            Image(
+                painter = painterResource(R.drawable.image),
+                contentDescription = "개 이미지",
+                modifier = Modifier
+                    .size(80.dp)
+                    .clip(CircleShape)
+            )
+
+        }
+        Text(
+            modifier = Modifier
+                .padding(top = 20.dp)
+                .fillMaxWidth()
+                .border(2.dp, color = Color.Black)
+                .padding(10.dp),
+
+            text = "Hello World!",
+            fontSize = 20.sp,
+            fontStyle = FontStyle.Normal,
+            fontWeight = FontWeight.Bold,
+            color = Color.Black,
+            textAlign = TextAlign.Start
+        )
+    }
+}
 
 @Composable
-fun Greeting(name: String, modifier: Modifier = Modifier) {
-    Text(
-        text = "Hello $name!",
-        modifier = modifier
-    )
+fun Profile_Card(modifier: Modifier = Modifier) {
+    Column(
+        modifier = modifier.padding(top = 45.dp)
+    ) {
+        SampleRow{
+            Box(
+                modifier = modifier.size(240.dp,140.dp)
+                    .fillMaxWidth()
+                    .border(
+                        color = Color.Black,
+                        width = 1.dp,
+                        shape = RoundedCornerShape(12.dp)
+                    )
+                    .padding(10.dp,)
+            )
+            {
+                Image(
+                    painter = painterResource(R.drawable.image2),
+                    contentDescription = null,
+                    modifier = Modifier
+                        .padding(start = 180.dp)
+                        .size(40.dp,40.dp)
```

Comment:
가로, 세로 길이가 모두 40.dp로 동일하다면
```suggestion
                  .size(40.dp)
```
이렇게 하나의 값만 넘겨주셔도 됩니다 ~